modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,143 @@
+** Manage {gitops-title} from a terminal.
+** Manage ArgoCD resources, such as `Applications`, `ApplicationSets`, `AppProjects`, `User accounts`, and `GPG keys` from a client terminal. link:https://issues.redhat.com/browse/GITOPS-3389[GITOPS-3389]
++
+[NOTE]
+====
+The `argocd` executable binary file is included in the archive and RPM formats.
+====
++
+[WARNING]
+==== 	
+The Argo CD CLI tool is a Technology Preview feature.
+====
+
+* With this update, `NotificationsConfiguration` custom resource (CR) is now supported. Before this update, you could update the `argocd-notifications-cm` ConfigMap to manage their templates, triggers, services and subscriptions. After this update, you cannot modify the `argocd-notifications-cm` ConfigMap. A new Custom Resource definition `NotificationsConfiguration` is introduced to manage the `argocd-notifications-cm` ConfigMap. link:https://issues.redhat.com/browse/GITOPS-4130[GITOPS-4130]
++
+[NOTE]
+====
+This update removes any configuration added to `argocd-notifications-cm` ConfigMap. Users must take a backup of their configuration and update that in the `default-notifications-configuration` custom resource of kind `NotificationsConfiguration` after upgrading to the new version. Also, with this update, any modifications to `argocd-notifications-cm` ConfigMap are not allowed.
+====
+
+* With this update, two new fields `.spec.applicationSet.sourceNamespaces` and `.spec.applicationSet.scmProviders` are introduced in the `ArgoCD` CRD to support `ApplicationSet` in any namespaces. Administrators can use these fields to define certain namespaces that manage `ApplicationSet` resources. link:https://issues.redhat.com/browse/GITOPS-3754[GITOPS-3754]
+
+* With this update, Argo CD server has the required permissions to manage `ApplicationSet` resources. link:https://issues.redhat.com/browse/GITOPS-3762[GITOPS-3762]
+
+* With this update, you can configure log levels, such as `debug`, `info`, `warn`, `error`, `panic` and `fatal`. The default log level set for the output is `info`. To change the log level, add the environment variable `LOG_LEVEL` in the `.spec.config.env` field of the `Subscription` CR. link:https://issues.redhat.com/browse/GITOPS-4016[GITOPS-4016]
++
+.Example output
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: gitops-operator
+  namespace: openshift-gitops-operator
+spec:
+  ...
+  config:
+    env:
+    - name: LOG_LEVEL
+      value: "error"
+----
+
+* With this update, Argo CD accepts the wildcard values in the `sourceNamespaces` field so that you can specify multiple namespaces or patterns of namespaces. To use this feature, specify the namespaces where Argo CD can manage applications in the `.spec.sourceNamespaces` field of the `ArgoCD` CR. link:https://issues.redhat.com/browse/GITOPS-3935[GITOPS-3935]
```

Comment:
```suggestion
* With this update, Argo CD accepts the wildcard values in the `sourceNamespaces` field so that you can specify multiple namespaces or patterns for namespaces. To use this feature, specify the namespaces where Argo CD can manage applications in the `.spec.sourceNamespaces` field of the `ArgoCD` CR. link:https://issues.redhat.com/browse/GITOPS-3935[GITOPS-3935]
```

Reply:
Fixed.